e2e/cypress/integration/specs/checkout/checkout-payments.b2c.e2e-spec.ts

```diff
@@ -33,13 +33,12 @@ describe('Checkout Payment', () => {
       LoginPage.navigateTo('/checkout/address');
```

Comment:
replace all (4) occurrences of .should('not.be.visible') by .should('not.exist') in this file - or do you leave this occurrences intentionally?

Reply:
Yes, the elements exist, I had to roll those back to 'not.be.visible' 😉